[2022-12-16 13:05:34,898] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: prepare_data.get_last_id scheduled__2022-12-16T12:46:00+00:00 [queued]>
[2022-12-16 13:05:35,092] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: prepare_data.get_last_id scheduled__2022-12-16T12:46:00+00:00 [queued]>
[2022-12-16 13:05:35,092] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-12-16 13:05:35,093] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2022-12-16 13:05:35,096] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-12-16 13:05:35,564] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): get_last_id> on 2022-12-16 12:46:00+00:00
[2022-12-16 13:05:35,686] {standard_task_runner.py:52} INFO - Started process 4225 to run task
[2022-12-16 13:05:35,819] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'prepare_data', 'get_last_id', 'scheduled__2022-12-16T12:46:00+00:00', '--job-id', '2083', '--raw', '--subdir', 'DAGS_FOLDER/prepare_data.py', '--cfg-path', '/tmp/tmpxlvhvcpp', '--error-file', '/tmp/tmp28q21p0m']
[2022-12-16 13:05:35,901] {standard_task_runner.py:80} INFO - Job 2083: Subtask get_last_id
[2022-12-16 13:05:36,912] {task_command.py:369} INFO - Running <TaskInstance: prepare_data.get_last_id scheduled__2022-12-16T12:46:00+00:00 [running]> on host 8e2f65ec4c74
[2022-12-16 13:05:38,017] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=prepare_data
AIRFLOW_CTX_TASK_ID=get_last_id
AIRFLOW_CTX_EXECUTION_DATE=2022-12-16T12:46:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-12-16T12:46:00+00:00
[2022-12-16 13:05:38,217] {base.py:68} INFO - Using connection ID 'gp_conn' for task execution.
[2022-12-16 13:05:38,494] {python.py:173} INFO - Done. Returned value was: None
[2022-12-16 13:05:38,711] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=prepare_data, task_id=get_last_id, execution_date=20221216T124600, start_date=20221216T130534, end_date=20221216T130538
[2022-12-16 13:05:38,930] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-12-16 13:05:39,465] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
